packages/website/src/snippets/index.tsx

```diff
@@ -1,55 +1,5 @@
-import { createBentoComponents } from "@buildo/bento-design-system";
+import { createBentoProvider } from "@buildo/bento-design-system";
 
 export * from "@buildo/bento-design-system";
 
-export const {
-  Actions,
-  AreaLoader,
-  Avatar,
-  Banner,
-  Breadcrumb,
-  Body,
-  Box,
-  Button,
-  Card,
-  Checkbox,
-  CheckboxField,
-  Chip,
-  Column,
-  Columns,
-  ContentBlock,
-  ContentWithSidebar,
-  DateField,
-  DesignSystemProvider,
-  Disclosure,
-  Feedback,
-  FileUploaderField,
-  Form,
-  FormSection,
-  FormRow,
-  Inline,
-  InlineLoader,
-  Inset,
-  List,
-  Menu,
-  Modal,
-  Navigation,
-  Placeholder,
-  NumberField,
-  ProgressBar,
-  RadioGroupField,
-  SearchBar,
-  SliderField,
-  Stack,
-  Stepper,
-  Switch,
-  Table,
-  tableColumn,
-  Tabs,
-  TextField,
-  TextArea,
-  Tiles,
-  Toast,
-  Tooltip,
-  SelectField,
-} = createBentoComponents();
+export const DesignSystemProvider = createBentoProvider();
```

Comment:
this was causing funny bugs all around the website; we should stop using `createBentoComponents`